aten/src/ATen/native/cuda/ForeachFunctors.cuh

```diff
@@ -656,6 +656,74 @@ struct PointwiseOpFunctor {
         }
 };
 
+template<typename T>
+struct BinaryOpListFunctor {
```

Comment:
I promise i will refactor the whole file one day.

Reply:
Ok, for now maybe one line comments for each variant would be helpful (unless you plan to refactor soon). 
E.g. BinaryOpListFunctor - 2 inputs, one out of place output, PointwiseOpFunctor - 3 inputs, one in-place output etc.

Reply:
ill refactor it soon